cev_eris.dme

```diff
@@ -856,8 +858,8 @@
 #include "code\game\objects\items\devices\organ_module\simple.dm"
 #include "code\game\objects\items\devices\organ_module\active\armblades.dm"
 #include "code\game\objects\items\devices\organ_module\active\engineer.dm"
-#include "code\game\objects\items\devices\organ_module\active\surgical.dm"

```

Comment:
How does this keep happening though?

Reply:
*Someone* edits dme by hand disregarding alphabetical order.

Reply:
Maybe older master branch

Reply:
I wish they would stop doing that.

Reply:
Well we could enforce it via CI so things like [this](https://github.com/discordia-space/CEV-Eris/pull/4223/commits/4e20a2621a80a1b3ed142546b0fb6c6b91d4eeec) do not happen.
Anyway, this is irrelevant to this PR.